File: nprime/nprime.py
import sys

def main():
    number = int(sys.argv[1])
    if number <= 1:
        print(2)
        return
    last_prime = 2
    prime_count = 1

    n = 3
    while True:
        is_prime = True
        for d in range(3, last_prime, 2):
            if n % d == 0:
                is_prime = False
                break
        if is_prime:
            last_prime = n
            prime_count += 1
            if prime_count >= number:
                print(last_prime)
                return
        n += 2

if __name__ == '__main__':
    main()

File: nprime/nprime.js

function main() {
  var number = parseInt(process.argv[2])
  if(number <= 1) {
    console.log(2);
    process.exit();
  }
  var last_prime = 2
  var prime_count = 1

  for(let n=3;;n+=2) {
    let is_prime = true;
    for(let d = 3; d<=last_prime;d+=2) {
      if(n % d == 0) {
        is_prime = false;
        break;
      }
    }
    if(is_prime) {
      last_prime = n;
      prime_count++;
      if(prime_count >= number) {
        console.log(last_prime);
        process.exit();
      }
    }
  }  
}

main()
